core/service/src/chain_spec.rs

```diff
@@ -159,7 +160,13 @@ impl<G: RuntimeGenesis> ChainSpec<G> {
 
 	/// Parse json content into a `ChainSpec`
 	pub fn from_embedded(json: &'static [u8]) -> Result<Self, String> {
```

Comment:
Why not drop this function?

Reply:
Backwards compatibility

Reply:
We don't have any released version, so just remove it.